examples/46_alter_column_drop_default.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "46_alter_column_drop_default",
+  "operations": [
+    {
+      "alter_column": {
+        "table": "tickets",
+        "column": "ticket_type",
+        "default": null,
+        "up": "",
+        "down": ""
```

The data migration in both directions can be a straight copy without modification:

```suggestion
        "up": "ticket_type",
        "down": "ticket_type"
```